cstore_fdw.h

```diff
@@ -170,6 +170,15 @@ typedef struct StripeSkipList
 } StripeSkipList;
 
 
+typedef struct BlockDataBuffers
+{
+	bool *existsArray;
+	Datum *valueArray;
+	StringInfo serializedValueBuffer;
```

Comment:
1. I think ```serializedValueBuffer``` is here because ```valueArray``` can have pointers to its buffer. So, I think the fact that this is result of serialization or not isn't important. So, can we rename this to ```valueBuffer```?

2. Can we put a comment here and explain why are we putting this here? It is not clear and can lead to confusion. (Our style for member field comments is: newline, then the comment, then the member field.)